src/mixer.c

```diff
@@ -545,15 +545,18 @@ void mixTable(void)
     }
 
     // motors for non-servo mixes
-    if (numberMotor > 1)
-        for (i = 0; i < numberMotor; i++)
+    if (numberMotor > 1) {
+        for (i = 0; i < numberMotor; i++) {
             motor[i] = rcCommand[THROTTLE] * currentMixer[i].throttle + axisPID[PITCH] * currentMixer[i].pitch + axisPID[ROLL] * currentMixer[i].roll + -cfg.yaw_direction * axisPID[YAW] * currentMixer[i].yaw;
-
-    if (f.FIXED_WING) {
-        if (!f.ARMED)
-            motor[0] = mcfg.mincommand; // Kill throttle when disarmed
-        else
-            motor[0] = constrain(rcCommand[THROTTLE], mcfg.minthrottle, mcfg.maxthrottle);
+            if (f.FIXED_WING) { // vector_trust handeling
+                if (cfg.fw_vector_trust) {
+                    if (f.PASSTHRU_MODE)
+                        motor[i] = rcCommand[THROTTLE] - rcCommand[YAW] * (i - 0.5f);
+                } else { // Override mixerTrustVector
+                    motor[i] = rcCommand[THROTTLE];
+                }
+            }
+        }
     }

```

Comment:
hmm, I'm actually not understanding why this is necessary?
also, is it vector TRUST or vector THRUST?
So if I understand this new code, the following happens:

1) for each motor, it writes motor[] = rcCommand[THROTTLE] * currentMixer[i].throttle + axisPID[PITCH] * currentMixer[i].pitch + axisPID[ROLL] * currentMixer[i].roll + -cfg.yaw_direction * axisPID[YAW] * currentMixer[i].yaw;
2) if it's in fixed wing mode, it replaces motor[] with either rccommand[throttle] or
3) If it's in fixed wing + "vector trust??", then it replaces with throttle + yaw formula.

Is this correct?

1) for each motor, it writes motor[] =